Realm.podspec

```diff
@@ -52,7 +52,8 @@ Pod::Spec.new do |s|
   s.osx.vendored_library    = 'core/libtightdb.a'
 
   s.subspec 'Headers' do |s|
-    s.source_files          = 'include/**/*.h'
+    s.source_files          = 'include/**/*.{h,hpp}'
```

Comment:
> It makes the podspec more complicated, not less

Ok true, this approach requires to explicitly listing the private header files. I agree with @tg the way it was before is better.